.travis.yml

```diff
@@ -1,12 +1,22 @@
 language: java
-sudo: false
+sudo: required
```

Comment:
can't we use this somehow? https://docs.travis-ci.com/user/migrating-from-legacy/#How-do-I-install-APT-sources-and-packages%3F